src/pages/my-courses/MyCourses.tsx

```diff
@@ -95,6 +95,45 @@ const MyCourses = () => {
     })
   }
 
+  const duplicateCourse = useCallback(
+    (id?: string) => {
+      const item = coursesResponse.items.find(
+        (element) => element._id === id
+      ) as Course
+
+      return CourseService.addCourse({
```

Comment:
can you for example only pass an item to add the course?

return CourseService.addCourse(item)

Reply:
Yes, you are right! Thanks.